modules/shared/components/atoms/ImageUpload/imageUpload.module.css

```diff
@@ -0,0 +1,17 @@
+.inputfile {
+  @apply opacity-0 w-full h-full absolute cursor-pointer;
+}
+.label {
+  @apply box-border bg-grey-shd7 rounded-md border-2 border-accent border-dashed 
+  flex flex-row justify-center items-center 
+  relative
+  w-auto
+  py-mx;
+}
+.text {
+  @apply text-accent text-sm font-medium not-italic;
+}
+
+.svg {
+  @apply mr-1;
+}
```

Comment:
please, consider using a more specific naming convention instead of these generic names for the classes for example ".text" could be ".img-upload-box-text" and ".svg" could be ".add-img-icon"